front/app/containers/Admin/projects/project/information/ReportRow.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+// components
+import { ListItem, Box, Title } from '@citizenlab/cl2-component-library';
+import ReportRowEditedText from 'containers/Admin/reporting/components/ReportBuilderPage/ReportRow/EditedText';
+import ReportRowButtons from 'containers/Admin/reporting/components/ReportBuilderPage/ReportRow/Buttons';
+
+// i18n
+import messages from './messages';
+import { useIntl } from 'utils/cl-intl';
+
+// utils
+import clHistory from 'utils/cl-router/history';
+
+import useDeleteReport from 'api/reports/useDeleteReport';
+import { Report } from 'api/reports/types';
+
+interface Props {
+  report: Report;
+}
+
+const ReportRow = ({ report }: Props) => {
+  const { mutate: deleteReport, isLoading } = useDeleteReport();
+  const { formatMessage } = useIntl();
+
+  const handleDeleteReport = async () => {
+    if (window.confirm(formatMessage(messages.areYouSureYouWantToDelete))) {
+      deleteReport(report.id);
+    }
+  };
+
+  // TODO link to correct instance
+  const reportPath = `/admin/reporting/report-builder/${report.id}`;
+
+  const handleEditReport = () => {
+    clHistory.push(`${reportPath}/editor`);
+  };
+
+  const handleViewReport = () => {
+    window.open(`${reportPath}/viewer`, '_blank');
+  };
```

Comment:
This code is the same in both Row components. Could we move it into the `ReportRowButtons` component to avoid duplication?

Reply:
I am not sure yet if we want to create a new report builder instance, or reuse the same report builder instance at this route. If we end up using the same one we can move this in there indeed